src/main/resources/beam-template.conf

```diff
@@ -726,6 +726,7 @@ beam.outputs.events.fileOutputFormats = "csv" # valid options: xml(.gz) , csv(.g
 # Events Writing Logging Levels:
 # Any event types not explicitly listed in overrideWritingLevels take on defaultWritingLevel
 beam.outputs.events.eventsToWrite = "ActivityEndEvent,ActivityStartEvent,PersonEntersVehicleEvent,PersonLeavesVehicleEvent,ModeChoiceEvent,PathTraversalEvent,ReserveRideHailEvent,ReplanningEvent,RefuelSessionEvent,ChargingPlugInEvent,ChargingPlugOutEvent,ParkingEvent,LeavingParkingEvent"
+beam.outputs.events.extraEventsToWrite: "string?"
```

Comment:
Should there be a default and/or add something to the docs so that this isn't a hidden feature?